geomesa-accumulo/geomesa-accumulo-spark-runtime/pom.xml

```diff
@@ -93,6 +93,10 @@
             <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
             <classifier>tests</classifier>
         </dependency>
+        <dependency>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-minicluster</artifactId>
+        </dependency>
```

Comment:
Just to make sure, this is test scope, right;)?

Reply:
yeah